fhir-search/src/main/java/com/ibm/fhir/search/location/bounding/BoundingMissing.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.search.location.bounding;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * This class is used to indicate a Missing search.
+ */
+public class BoundingMissing extends Bounding {
```

Comment:
Is this class needed?  Why isn't near:missing handled similarly to the other search parameters?

Reply:
it is.  this is used to signal the fact that it doesn't need the 4 parameters or more related to a bounding area.